build_script.sh

```diff
@@ -9,4 +18,17 @@ fi
 
 cd build
 cmake ..
-cmake --build . -j$CPUS
\ No newline at end of file
+cmake --build . -j$CPUS
+
+cd "${topDir}"
+cp -f build/src/pythonmonkey.so python/pythonmonkey/
```

Comment:
This is supposed to be done in https://github.com/Distributive-Network/PythonMonkey/blob/dce60a/build.py#L29-L34